# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/hotfix-arm.patch
# Copyright (C) 2022 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-122.0/media/ffvpx/libavcodec/moz.build.vanilla	2024-01-28 19:09:37.685841346 +0100
+++ firefox-122.0/media/ffvpx/libavcodec/moz.build	2024-01-28 19:09:43.298841122 +0100
@@ -9,8 +9,6 @@
 if CONFIG['FFVPX_ASFLAGS']:
     if CONFIG['TARGET_CPU'] == 'x86' or CONFIG['TARGET_CPU'] == 'x86_64':
         DIRS += ['x86']
-    elif CONFIG['TARGET_CPU'] == 'arm':
-        DIRS += ['arm']
 
 if CONFIG['TARGET_CPU'] == 'aarch64':
     DIRS += ['aarch64']
--- firefox-122.0/media/ffvpx/libavutil/moz.build.vanilla	2024-01-28 19:09:37.688841346 +0100
+++ firefox-122.0/media/ffvpx/libavutil/moz.build	2024-01-28 19:09:47.914840938 +0100
@@ -9,8 +9,6 @@
 if CONFIG['FFVPX_ASFLAGS']:
     if CONFIG['TARGET_CPU'] == 'x86' or CONFIG['TARGET_CPU'] == 'x86_64':
         DIRS += ['x86']
-    elif CONFIG['TARGET_CPU'] == 'arm':
-        DIRS += ['arm']
 
 if CONFIG['TARGET_CPU'] == 'aarch64':
     DIRS += ['aarch64']
